cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -37,6 +37,11 @@ message OptionalMap {
     map<string, string> data = 1;
 }
 
+// Used in edit messages to distinguish between empty strings and missing strings
+message OptionalString {
```

Comment:
this can go now ;)

Reply:
i thought i deleted that.. i guess i didnt

Reply:
i never hit cmd s